core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/SenderThreadLocalBackOffData.java

```diff
@@ -68,6 +68,15 @@ public SenderThreadLocalBackOffData(long[] backOffTimeoutsInMillis, long addMill
     public boolean isTryingToSend() {
         return currentBackOffIndex != -1;
     }
+    
```

Comment:
Missing Java docs

Reply:
Not needed, removed

Reply:
Removed method, not used anywhere provided no value

Reply:
Okay